keyboards/merge/uma/readme.md

```diff
@@ -0,0 +1,14 @@
+# UM-70
+
+![UM-A](https://i.imgur.com/Bm1mRDbl.jpg)
+
+A Alice like layout keyboard with arrow keys, encoder and OLED screen sold by Merge. [Product page](https://mergedesign.store/products/um-70)
+
+* Keyboard Maintainer: [duoshock](https://github.com/duoshock)
+* Hardware Availability: [Merge Store](https://mergedesign.store/products/um-a)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Sure thing. I've just updated it. Thanks!